revscoring/utilities/tests/test_union_merge_observations.py

```diff
@@ -0,0 +1,48 @@
+import json
+import os.path
+import tempfile
+
+import revscoring.utilities.union_merge_observations
+
+
+def test_union():
+    # Get test fixtures.
+    data_dir = os.path.dirname(__file__) + "/data"
+    in_files = [
+        data_dir + "/labeled_revisions.json",
+        data_dir + "/labeled_foo.json",
+    ]
+
+    # Receive the output in a temporary file.
+    (fh, out_file) = tempfile.mkstemp()
```

Comment:
I see.  Maybe it would be better to test these things separately -- *i.e.*, in units.  I guess the reason I keep pushing on this is because it's working hard read actual files when the real thing we want to test is the merge and union functionality.  This feels like an integration test.